Classes/EventStore/EventStore.php

```diff
@@ -15,66 +15,29 @@
 use Neos\Cqrs\EventStore\Exception\EventStreamNotFoundException;
 use Neos\Cqrs\EventStore\Storage\EventStorageInterface;
 use TYPO3\Flow\Annotations as Flow;
-use TYPO3\Flow\Log\SystemLoggerInterface;
 
 /**
- * EventStore
+ * @Flow\Scope("singleton")
```

Comment:
Depending on the outcome of neos/flow-development-collection#671 this might become a prototype again